docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -0,0 +1,1050 @@
+{
```

Comment:
<p>Just one thing: Sorry my previous review comment was confusing: there needs to be only one <code>#</code> before the header for the title of the notebook so here it should be <code># Sharpe Ratio Maximization</code> instead of <code>## Sharpe...</code> .</p><p> </p><p>The reason is if you look in the sidebar when you build the docs, on the index page you only see the title of the notebook (if you were to click on the link to view the notebook, you would see the subheaders in the expandable section under Sharpe Ratio Maximization in the sidebar).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>